spec/models/seed_spec.rb

```diff
@@ -152,4 +152,32 @@
     end
   end
 
+  context 'photos' do
+    before(:each) do
+      @seed = FactoryGirl.create(:seed)
+      @photo = FactoryGirl.create(:photo)
```

Comment:
Is it possible to just define the association in the call to create? Something like:

`@photo = FactoryGirl.create(:photo, :seed => @seed)`